src/preferences/dialog/dlgprefcolors.cpp

```diff
@@ -84,31 +90,39 @@ DlgPrefColors::~DlgPrefColors() {
 void DlgPrefColors::slotUpdate() {
     comboBoxHotcueColors->clear();
     comboBoxTrackColors->clear();
+    comboBoxKeyColors->clear();
     checkboxKeyColorsEnabled->setChecked(
             m_pConfig->getValue(kKeyColorsEnabledConfigKey,
                     BaseTrackTableModel::kKeyColorsEnabledDefault));
     for (const auto& palette : std::as_const(mixxx::PredefinedColorPalettes::kPalettes)) {
         QString paletteName = palette.getName();
         QIcon paletteIcon = drawPalettePreview(paletteName);
-        comboBoxHotcueColors->addItem(paletteName);
+        comboBoxHotcueColors->addItem(tr(qPrintable(paletteName)), paletteName);
```

Comment:
Did you consider to make paletteName `const char*` to get around the QString roundtrip?

Reply:
Didn't think of that, but sounds like a good idea. I will give that a shot!

Reply:
Done, this looks nicer now.

Reply:
Wait. I just broke some of the palette names, will have to see why.

![image](https://github.com/user-attachments/assets/42448478-c04b-401d-afa2-f8afcf68763e)

Reply:
Interestingly, the palette names break differently each time I restart. 
![image](https://github.com/user-attachments/assets/4b8d6c0b-50ee-44c9-9ec1-35f6f3c5b382)
![image](https://github.com/user-attachments/assets/eff57cf0-9232-4be5-8ec7-7b8615a4e400)